### DISCUSSION TITLE
Delete just one or more episode or season file from series / anime

### DISCUSSION THREAD

**Discussion by Nibeck1309** [pymedusa/Medusa#5235] [Ideas & Feature requests]
Hello,

With Medusa we can delete a complete series as well as the files.
But we can not just delete one or more episodes  or season of a series / animes.

the goal is to free space on the hard disk.

When deleting one or more episodes, the status would change from download to archive.

Would it be possible to implement this posibility?

Sincerely

**Comment by natiz** [comment]
+1 would love this

**Comment by p0psicles** [comment]
So you want to delete the file(s) for an episode from disk? But keep the episode in the library?

**Comment by natiz** [comment]
usually what I do is delete episode from disk, then mark it as archived
